installer/arch/prepare

```diff
@@ -0,0 +1,244 @@
+#!/bin/sh -e
+# Copyright (c) 2014 The crouton Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# This is a distro-specific continuation of the prepare.sh script.
+
+PKGEXT='tar.xz'
+DISTROAKA='arch'
+
+# aurcompile <package> <extra makepkg options>: compile package from AUR.
```

Comment:
`sbc` is not actually needed for CRAS plugin operation (see #1353). I'll remove the function.